database/src/migrations/20230414000001_tr_project_funding_source.ts

```diff
@@ -0,0 +1,60 @@
+import { Knex } from 'knex';
+
+/**
+ * Apply biohub release changes.
+ *
+ * @export
+ * @param {Knex} knex
+ * @return {*}  {Promise<void>}
+ */
+export async function up(knex: Knex): Promise<void> {
+  await knex.raw(`
+  set search_path=biohub;
+
+  create or replace function tr_project_funding_source() returns trigger
+  language plpgsql
+  security invoker
+  as
+  $$
+  -- *******************************************************************
+  -- Procedure: tr_project_funding_source
+  -- Purpose: performs specific data validation
+  --
+  -- MODIFICATION HISTORY
+  -- Person           Date        Comments
+  -- ---------------- ----------- --------------------------------------
+  -- charlie.garrettjones@quartech.com
+  --                  2021-01-03  initial release
+  -- charlie.garrettjones@quartech.com
+  --                  2023-04-17  BHBC-2297 adjustment to accommodate first nations
+  -- *******************************************************************
```

Comment:
Could add an entry to this table explaining what has changed

Reply:
Charlie has a comment on it, he also has a draft PR with the migration changes (I just branched off), I could ask him to add a few more details though as it is a little confusing.

Reply:
If Charlie is on it, I'm not worried 😌